tools/Test/Logon.ps1

```diff
@@ -0,0 +1,17 @@
+[CmdletBinding()]
```

Comment:
Deleted the file.

From: Mark Cowlishaw [mailto:notifications@github.com]
Sent: Tuesday, May 24, 2016 6:05 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Iouri Simernitski <iouris@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Azure/azure-powershell] Commiting DevTestLabs powershell cmdlets (#2293)


In tools/Test/Logon.ps1<https://github.com/Azure/azure-powershell/pull/2293#discussion_r64501047>:

> @@ -0,0 +1,17 @@

> +[CmdletBinding()]

@iouri-s<https://github.com/iouri-s> don't see this change, please update.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/2293/files/b4a768078f30037ef784ed0ba456f2f5d5b3d2b4#r64501047>